armsrc/iso14443a.c

```diff
@@ -2535,6 +2541,11 @@ void Mifare1ksim(uint8_t flags, uint8_t exitAfterNReads, uint8_t arg2, uint8_t *
 			LED_C_OFF();
 			crypto1_destroy(pcs);
 			cardAUTHKEY = 0xff;
+			if (flags & FLAG_RANDOM_NONCE) {
+				nonce = prand();
+			} else {
+				nonce++;
```

Comment:
Looks like leaving the code as-is makes it even _easier_ to detect PM3 -- it will have a fixed nonce of `01020304` until you attempt to authenticate. If you're doing Moebius attack, it'll then move to a nonce of `70e151c`. I'll modify the code to remove the increment, and add a note that it breaks the standard reader attack.